README.md

```diff
@@ -47,3 +47,10 @@ this Program grant you additional permission to convey the resulting work.
 Library | Group ID | Artifact ID | SPDX License ID
 :-- | :-- | :-- | :--
 Jakarta Mail | com.sun.mail | jakarta.mail | GPL-2.0-only
+
+## Open Source Delevoper Licence for YourKit Profiler [![YourKit logo](https://www.yourkit.com/images/yklogo.png)]
+
+YourKit supports open source projects with innovative and intelligent tools for monitoring and profiling Java and .NET applications.
+YourKit is the creator of <a href="https://www.yourkit.com/java/profiler/">YourKit Java Profiler</a>, <a href="https://www.yourkit.com/.net/profiler/">YourKit .NET Profiler</a>, and <a href="https://www.yourkit.com/youmonitor/">YourKit YouMonitor</a>.
+
+YourKit grants the TripleA development project an open source licence for YourKit Java Profiler.
```

Comment:
Grammar nit:

```
YourKit grants the TripleA development
```
->
```
 [YourKit](https://www.yourkit.com/) has graciously granted an open source license of [YourKit Java Profiler](https://www.yourkit.com/java/profiler/) to TripleA. 
```

WDYT, reads better?

Reply:
It's all fine by me. If you remove the example from YourKit, then make sure to include a link to Yourkit.com in the remaining text. More important is to add the same text to the start page of www.triplea-game.org. How can we make that happen?